setup/scripts/run_docker.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
```

Comment:
Perhaps it's not a big deal, but the file naming here is not consistent.  Do you want to make it so as `docker_runner.sh`?

Reply:
Reverted to your original suggestion: `run_docker.sh` matching the naming convetion of the other scripts (verb_noun.sh)